can_tools/scrapers/official/HI/hi_county.py

```diff
@@ -0,0 +1,88 @@
+import pandas as pd
+import us
+import requests
+import re
+import json
+from bs4 import BeautifulSoup
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import (
+    TableauDashboard,
+    TableauMapClick,
+    StateDashboard,
+)
+
+from can_tools.scrapers.util import requests_retry_session
+
+
+class HawaiiVaccineCounty(TableauDashboard, StateDashboard):
```

Comment:
TableauDashboard is a subclass of StateDashboard, so this line can be:

```python
class HawaiiVaccineCounty(TableauDashboard):
```

Reply:
Removed `StateDashboard`